main.py

```diff
@@ -36,20 +37,27 @@
 print_markdown(
     "### Thanks for using this tool! [Feel free to contribute to this project on GitHub!](https://lewismenelaws.com) If you have any questions, feel free to reach out to me on Twitter or submit a GitHub issue. You can find solutions to many common problems in the [Documentation](https://luka-hietala.gitbook.io/documentation-for-the-reddit-bot/)"
 )
-print_step(f"You are using v{__VERSION__} of the bot")
+print_step(f"You are using v{__VERSION__} of the bot in the {__BRANCH__} branch.")
 
 
 def main(POST_ID=None):
-    cleanup()
     reddit_object = get_subreddit_threads(POST_ID)
+    id(reddit_object)
     length, number_of_comments = save_text_to_mp3(reddit_object)
     length = math.ceil(length)
     download_screenshots_of_reddit_posts(reddit_object, number_of_comments)
     bg_config = get_background_config()
     download_background(bg_config)
-    chop_background_video(bg_config, length)
+    chop_background_video(bg_config, length, reddit_object)
     make_final_video(number_of_comments, length, reddit_object, bg_config)
 
+def id(reddit_obj: dict):
```

Comment:
This function should probably placed in the utils folder, and omitting the print statement

All future needs for the id can then use this function

Reply:
I have put it in the utils folder